js/views/modals/Settings/Store.js

```diff
@@ -11,74 +10,74 @@ import { openSimpleMessage } from '../SimpleMessage';
 
 export default class extends baseVw {
   constructor(options = {}) {
-    super({
+    const opts = {
       className: 'settingsStore',
       ...options,
-    });
+      initialState: {
```

Comment:
I don't think you want to use the setState functionality here. It doesn't really play nice with forms since it triggers a full re-render and existing form data will be lost. For example, put a peerId in the Find text field and then check/uncheck the Verified Mods filter. You'll lose that peer id. It's just way too heavy handed to re-render this entire store view.

The setState approach also generally doesn't play well with views that modify their state via jQuery (like this one). You really want to pick a path and either go the jQuery approach or setState approach. For example, if you saved this tab and the save was taking a while and you check/uncheck the Verified Mods filter, you'd lose the "processing" state of the Save button.

As far as handling the `showVerifiedOnly`, you'd want to do that in a way in which it just reloads the available moderators and doesn't trigger a full re-render of everything else since this view wasn't structured in a way to re-create it's state on re-renders.

(dead horse alert... without a virtual dom to help us to only selectively re-render specific parts that change, it's a fine game between using a jQuery / setState approach and/or being much more granular with your sub views.)